scripts/parachain/local.sh

```diff
@@ -91,13 +90,22 @@ start_collator() {
     --execution=wasm \
     --port=$relay_chain_port \
     --rpc-port=$relay_chain_rpc_port \
-    --tmp \
     --ws-port=$relay_chain_ws_port \
     -lcumulus_collator=trace \
-    -lruntime=trace
+    -lruntime=trace"
+    
+    $LAUNCH_PARACHAIN_CMD & node_pid=$!
+
+    sleep 30
+    DATA='{ "id":1, "jsonrpc":"2.0", "method":"author_insertKey", "params":["nmbs", "'"$seed"'", "'"$public_key"'"] }'
+    curl -H 'Content-Type: application/json' --data "$DATA" localhost:$collator_rpc_port
+
+    sleep 5
+    kill $node_pid
+    sleep 5
+    $LAUNCH_PARACHAIN_CMD
```

Comment:
This part of the script waits some time to let to node start up and then inserts the provided `$seed` and `$public key` parameters into the collator's keystore. The node is stopped and finally restarted again.
It is possible to see that the keys were really added by querying `author_hasKeys`.

Keys were also added manually through `apps`  but the end result was the same.